nuxeo-coldstorage-package/src/main/resources/install/templates/coldstorage/nxserver/config/s3-glacier-blob-provider-config.xml.nxftl

```diff
@@ -6,15 +6,15 @@
             <property name="awsid">${nuxeo.aws.accessKeyId}</property>
             <property name="awssecret">${nuxeo.aws.secretKey}</property>
             <property name="region">${nuxeo.aws.region}</property>
-            <property name="bucket">${nuxeo.glacier.bucket}</property>
-            <property name="bucket_prefix">${nuxeo.glacier.bucket_prefix}/</property>
+            <property name="bucket">${coldstorage.bucket}</property>
+            <property name="bucket_prefix">${coldstorage.bucket_prefix}/</property>
             <property name="directdownload">true</property>
             <property name="directdownload.expire">3600</property>
-            <property name="cachesize">100MB</property>
-            <property name="connection.max">50</property>
-            <property name="connection.retry">3</property>
-            <property name="connection.timeout">50000</property>
-            <property name="socket.timeout">50000</property>
+            <property name="cachesize">${nuxeo.s3storage.cachesize}</property>
+            <property name="connection.max">${nuxeo.s3storage.connection.max}</property>
+            <property name="connection.retry">${nuxeo.s3storage.connection.retry}</property>
+            <property name="connection.timeout">${nuxeo.s3storage.connection.timeout}</property>
+            <property name="socket.timeout">${nuxeo.s3storage.socket.timeout}</property>
```

Comment:
I am wondering if we have to provide the `key strategy` as we did on Aspera [Here](https://github.com/nuxeo/nuxeo-aspera-connector/blob/10.10/nuxeo-aspera-package/src/main/resources/install/templates/aspera/nxserver/config/nuxeo-aspera-config.xml.nxftl#L62). I think we can extract it as a variable and put a default variable. which will give us more flexibility